### DISCUSSION TITLE
3.3.0 RC5

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#6249] [Design & tech]
## What's Changed
* Call async methods when in an async method by @SergerGood in https://github.com/elsa-workflows/elsa-core/pull/6218
* Drop .NET 6 Support by @sfmskywalker in https://github.com/elsa-workflows/elsa-core/pull/6233
* Added example of perfomance test by @SergerGood in https://github.com/elsa-workflows/elsa-core/pull/6244
* Add support for ExpandoObject deserialization in ObjectConverter by @sfmskywalker in https://github.com/elsa-workflows/elsa-core/pull/6248

## New Contributors
* @SergerGood made their first contribution in https://github.com/elsa-workflows/elsa-core/pull/6218

**Full Changelog**: https://github.com/elsa-workflows/elsa-core/compare/3.3.0-rc4...3.3.0-rc5

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-core/releases/tag/3.3.0-rc5'>3.3.0 RC5</a>.</em>

**Comment by onerkaya** [comment]
hi @sfmskywalker , 

I submit new bug > https://github.com/elsa-workflows/elsa-core/issues/6262 

After updating from packet 3.2.1 to 3.3.0 RC5, the Hangfire call to the ResumeWorkflowJob.ExecuteAsync method fails because the tenantID is null when multi-tenancy is activated using elsa.UseTenants()

**Reply by sfmskywalker** [reply]
Thanks for letting me know 👍🏻

**Comment by sfmskywalker** [comment]
Closing this historical pre-release announcement. The reported Hangfire multi-tenancy problem was tracked separately in [#6262](https://github.com/elsa-workflows/elsa-core/issues/6262), which is closed; please open a current issue with a minimal reproduction if the problem persists on a supported release.
